src/server.ts

```diff
@@ -1,5 +1,9 @@
 "use strict";
 import { app } from "./routes";
+import { connectMongo } from "./services/mongoDb";
 const PORT: number | string = process.env.PORT || 3001;
+const MONGODB_URI =
+  "mongodb+srv://fonysony:6GyEGuDGRQP5Nvsh@octo-spork.pgoj3jj.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
Need to swap this out for an environment variable (i.e. process.env.MONGODB_URI) so it can be tested locally plus the server can be freely pointed at any given Atlas db without exposing secrets, passwords, etc